gha-webhook-listener.py

```diff
@@ -291,6 +300,12 @@ def tidy_extract_directory(target_dir, cleanup_dir, versions_to_keep):
         ), required=True,
     )
 
+    parser.add_argument(
+        "--branch-name", dest="branch_name", default="master", help=(
+            "Name of the branch to deploy from."
+        ), required=True,
```

Comment:
what does it mean to set `required` when there is a default?

Reply:
It means I’m a script kiddie and copied from the wrong lines.